packages/cli/src/decorators/Redactable.ts

```diff
@@ -0,0 +1,40 @@
+import type { UserLike } from '@/eventbus/audit.types';
+
+function toRedactable(userLike: UserLike) {
+	return {
+		userId: userLike.id,
+		_email: userLike.email,
+		_firstName: userLike.firstName,
+		_lastName: userLike.lastName,
+		globalRole: userLike.role,
+	};
+}
+
+type FieldName = 'user' | 'inviter' | 'invitee';
+
+/**
+ * Mark redactable properties in a `UserLike` field in an `AuditorService`
+ * method arg. These properties will be later redacted by the log streaming
+ * destination based on user prefs. Relevant only for `n8n.audit.*` logs.
```

Comment:
Nice description 👍 An `@example` would make it easier to understand how the function works

Reply:
Expanded docline